scripts/worker.py

```diff
@@ -215,42 +216,40 @@ def train_worker(gpu, ngpu_per_node, args, timers=None):
         train(rank=rank, gpu=gpu, task=args.task, model=model,
               train_loader=train_loader,
               loss_func=loss_func, optimizer=optimizer, epoch=epoch,
-              epochs=args.epochs, clip_grad=args.clip_grad,
```

Comment:
Why not remove the clip_grad argument from this function altogether?

Reply:
good point, can do that.

Reply:
done